src/Dierckx.jl

```diff
@@ -952,10 +952,10 @@ function evaluate(spline::Spline2D, x::AbstractVector, y::AbstractVector)
     return z
 end
 
-function evaluate(spline::Spline2D, x::Real, y::Real)
+function evaluate!(wrk::Vector{Float64}, Vspline::Spline2D, x::Real, y::Real)
     ier = Ref{Int32}()
     lwrk = spline.kx + spline.ky + 2
```

Comment:
This `spline` is undefined.  Was CI not run on this PR.

Reply:
Looks like a typo - the argument passed to `evaluate!` should probably be changed.